packages/zcli-apps/src/commands/apps/server.ts

```diff
@@ -53,14 +54,38 @@ export default class Server extends Command {
       res.end(JSON.stringify(appJSON))
     })
 
-    appJSON.installations.forEach((installation: Installation, index: number) => {
-      app.use(`/${installation.app_id}/assets`, express.static(`${appPaths[index]}/assets`))
-    })
+    const setAppAssetsMiddleware = () => {
+      appJSON.installations.forEach((installation: Installation, index: number) => {
+        app.use(`/${installation.app_id}/assets`, express.static(`${appPaths[index]}/assets`))
+      })
+    }
+
+    // Keep references of watchers for unwatching later
+    const watchers = appPaths.map(appPath =>
+      fs.watch(appPath, async (eventType, filename) => {
```

Comment:
Overall looks great. Few questions,

1. Any specific reason we are not using cross-platform lib like Chokidar here? I think it solves many inconsistencies between various platforms.

2. Also, should we consider ignoring `node_modules` directory? I think it might be a good idea to ignore that to avoid any potential perf issues (recursive nature of node_modules, and can sometimes be millions of files)

Reply:
Chokidar was used at first I believe but tests failed on Linux and Windows.

Reply:
1. Yup, we tried Chokidar but [tests failed on Linux and Windows](https://github.com/zendesk/zcli/actions/runs/1460230663). 
2. What do you mean by ignoring `node_modules` directory? It's in `.gitignore`.
Thanks

Reply:
@etanxing not our `node_modules`, 3rd party apps `node_modules`.

Do we know why chokidar failed? could be a real test failure or pointing to some flaky tests.

Reply:
The chokidar watcher wasn’t triggered when `manifest.json` was changed on Linux and Windows. On MacOS, the tests succeed so there is a cross platform issue with chokidar.